### PR TITLE
DOCS-691: Update Cloud sync with deletion note

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/services/data/configure-cloud-sync.md [hunk 1]
```diff
@@ -39,6 +39,12 @@ To change the sync interval, specify an interval in minutes in the interval fiel
 
 {{% /expand%}}
 
+### Pause sync
+
+You can pause Cloud Sync at any time by navigating to the **Services** tab on your robot's **config** tab and disabling **Syncing** for your [Data Management Service](../).
+If you have captured data that you do not want to sync, delete the data on the robot before resuming Cloud Sync.
+To delete the data locally, `ssh` into your robot and delete the data in the directory where you capture data.
```
Comment (agreenb) [inline docs/services/data/configure-cloud-sync.md:46]: In the above JSON on line 33, could you remove the `capture_dir` field or set it an empty "" value until we change this to handle `~/.viam/capture`?

Perhaps a note here, in the meantime, that if `capture_dir` is unspecified it'll be saved in ~/.viam/capture

Comment (npentrel) [reply docs/services/data/configure-cloud-sync.md:50]: done!

**Review by agreenb — COMMENTED** [review]
So sorry - while you're updating here, could we update all references to /.viam/capture to ~/.viam/capture in Configure Cloud Sync and Configure Data Capture? If you'd prefer to do that in a separate DOCS PR, happy to make a ticket for it.

**Comment by npentrel** [comment]
Let's do that in https://github.com/viamrobotics/docs/pull/1112 @agreenb

**Review by agreenb — APPROVED** [review]
Thank you!

**Pull request merged**